### PR TITLE
Merge release/5.0.0 back to dev

### PR DESCRIPTION
**Pull request by gitaumoses4**

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
+def common4jVersion = "2.0.0"
```
Comment (p3dr0rv) [inline common/build.gradle]: should we restore this value?

**Comment by p3dr0rv** [comment]
are we removing this yubikit-related code from dev? @melissaahn @gitaumoses4

File: common/build.gradle (hunk 1 above)
Comment (shahzaibj) [reply common/build.gradle]: We should definitely restore this. Good catch @p3dr0rv

**Comment by shahzaibj** [comment]
> are we removing this yubikit-related code from dev? @melissaahn @gitaumoses4

I don't think we want to remove this.

**Comment by melissaahn** [comment]
Added the YubiKit changes back in.

File: changelog.txt [hunk 2]
```diff
@@ -2,14 +2,19 @@ V.Next
...
 - [MINOR] Support WordApp local apk install in UI automation flows (#1732)
-- [PATCH] Add exception handling in Content Provider strategy, for broker communication (#1722)
-- [MINOR] Support TenantID value from eSTS in PKeyAuth flows (#1712)
-- [MINOR] Implement cert loader for both multiple and legacy WPJ data store in PKeyAuth (#1711)
 - [PATCH] Fix SDK Cancel when using authorization in current task (#1743)
 - [MINOR] Added support for broadcasting to applications installed on the device. (#1744)
 - [PATCH] Fix APPLICATION_CANCELLED by handling back button press (#1725)
 - [PATCH] Added Base64 encode for the AuthorizationRequest state parameter (#1750)
 
+Version 5.0.0
```
Comment (iamgusain) [inline changelog.txt:12]: Can you also add an entry here for [4.2.0 release](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/release/4.2.0/changelog.txt), and then update the changes in 5.0.0

Version 4.2.0
----------
- [PATCH] Add exception handling in Content Provider strategy, for broker communication (#1722)
- [MINOR] Support TenantID value from eSTS in PKeyAuth flows (#1712)
- [MINOR] Implement cert loader for both multiple and legacy WPJ data store in PKeyAuth (#1711)

**Comment by melissaahn** [comment]
I'm going to remove the YubiKit changes again. Going to keep YubiKit features separate from Dev until August.

File: common/build.gradle [hunk 3]
```diff
@@ -28,7 +28,7 @@ codeCoverageReport {
     coverage.enabled = enableCodeCoverage
 }
 
-def common4jVersion = "0.0.+"
+def common4jVersion = "0.0.*"
```
Comment (gitaumoses4) [inline common/build.gradle]:
```suggestion
def common4jVersion = "0.0.+"
```

**Review by p3dr0rv — APPROVED** [review]
LGTM, Thanks Moses

**Review by iamgusain — APPROVED** [review]

**Review by melissaahn — APPROVED** [review]

**Pull request merged**